### ISSUE TITLE
Bambu slicer load filament issue

### ISSUE THREAD

**Issue by Crobilt** [bambulab/BambuStudio#2720]
### Bambu Studio Version

1.7.7.89

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11

### Printer

Bambu P1S

### How to reproduce

When I push the load button in Bambu slicer it doesn't do anything. Unload button works fine

### Actual results

Nothing

### Expected results

Filament should load

### Project file & Debug log uploads

no file available

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by lavinir** [comment]
Was anything ever done with this? I seem to be having the same issue

**Comment by Crobilt** [comment]
No. I still can not unload filament. 🦆 Cro On Mar 16, 2024, at 14:13, lavinir ***@***.***> wrote:﻿
Was anything ever done with this? I seem to be having the same issue
